distribution/docs/src/main/jdocs/content/_tables/Csw_Registry_Store-table-contents.adoc

```diff
@@ -1,5 +1,9 @@
-:type: documentation
+:title: CSW Registry Store
+:id: Csw_Registry_Store
+:type: table
 :status: published
+:application: ${ddf-registry}
+:summary: Registry CSW Store.
```

Comment:
That's fine. I just thought that the file name, title, and description should match.